### PR TITLE
sanctuary-scripts@^3.1.0

### PR DESCRIPTION
**Pull request by davidchambers**
Once released, `sanctuary-scripts@3.1.0` will address sanctuary-js/sanctuary-scripts#14.

I am opening this pull request, which currently depends on an unreleased commit, to demonstrate that the `lint-readme` changes in the forthcoming sanctuary-scripts pull request have the intended effect.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -8,13 +8,9 @@
     {
       "files": ["*.md"],
       "plugins": ["markdown"],
-      "env": {"es6": true},
```
Comment (davidchambers) [inline .eslintrc.json:11]: This is inherited.

File: .eslintrc.json [hunk 2]
```diff
@@ -8,13 +8,9 @@
...
       "globals": {"checkPermission": false, "findRoles": false},
       "rules": {
-        "strict": ["off"],
```
Comment (davidchambers) [inline .eslintrc.json:14]: This is inherited.

File: .eslintrc.json [hunk 3]
```diff
@@ -8,13 +8,9 @@
...
-        "no-extra-semi": ["off"],
-        "semi-style": ["off"],
-        "semi": ["off"]
+        "no-unused-vars": ["error", {"varsIgnorePattern": "^(checkPermission|findRoles)$"}]
```
Comment (davidchambers) [inline .eslintrc.json:12]: I prefer this to `<!-- eslint-disable no-unused-vars -->`, but I'm happy to revert this change if you have a different preference.

**Comment by Avaq** [comment]
It seems that the "draft PR" feature prevents the pull request build from triggering. We would want the build to trigger to observe the effects of these changes.

**Comment by davidchambers** [comment]
I marked the pull request as ready for review, although it is not yet ready to merge. It looks as though I need to push another commit to trigger Travis.

**Comment by davidchambers** [comment]
<https://github.com/sanctuary-js/sanctuary-scripts/releases/tag/v3.1.0>

**Comment by Avaq** [comment]
Awesome!
